src/rsatoolbox/rdm/calc_unbalanced.py

```diff
@@ -199,193 +181,3 @@ def calc_one_similarity(data_i, data_j,
         method_idx, noise=noise,
         prior_lambda=prior_lambda, prior_weight=prior_weight,
         weighting=weight_idx)
-
-
-def calc_one_dissimilarity_cv(dataset, descriptor, i_des, j_des,
```

Comment:
@HeikoSchuett could you check that these are the functions that you meant for us to remove?

Reply:
I'll just go ahead with this for now and we can always restore them if I went too far